apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V80__new_schema_beneficiaries.sql

```diff
@@ -0,0 +1,727 @@
+-- NEW_SCHEMA_BENEFICIARIES.SQL
+-- flyway migration for BENEFICIARIES table into
+-- a new schema structure that:
+--   1) changes data type of BENE_ID from varchar to BIGINT.
+--
+-- Once current table data is migrated to new table name/structure, a 
+-- subsequent PR will be deployed that changes the ORM model(s) and
+-- operational code for Beneficiaries table.
```

Comment:
I think this isn't true anymore right? Since this is a combo code/migration PR now that we have the migrator.